app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -554,55 +342,8 @@ private void handleItemEvents(final ChatViewHolder chatViewHolder, final int pos
      * @param position position of view
      */
     private void handleItemEvents(final MapViewHolder mapViewHolder, final int position) {
-        final ChatMessage model = getData().get(position);
+        mapViewHolder.setView(getData().get(position), currContext, ((MainActivity)currContext).recyclerAdapter, position);
```

Comment:
I think just passing `this` instead of `((MainActivity)currContext).recyclerAdapter` should work.